llm-examples/mpt/mpt-7b/01_load_inference.py

```diff
@@ -73,5 +85,22 @@ def generate_text(prompt, **kwargs):
 
 # COMMAND ----------
 
+generate_kwargs = {
+  'max_new_tokens': 100,
+  'temperature': 0.2,
+  'top_p': 0.9,
+  'top_k': 50,
+  'repetition_penalty': 1.0,
+  'no_repeat_ngram_size': 0,
+  'use_cache': False,
```

Comment:
Is there a benefit for setting it as False? With the default True it should result in faster decoding: https://huggingface.co/docs/transformers/main_classes/text_generation#transformers.GenerationConfig